tests/characters/test_bob_joins_policy_and_retrieves.py

```diff
@@ -100,7 +101,15 @@ def test_bob_joins_policy_and_retrieves(federated_alice,
 
     assert plaintext == delivered_cleartexts[0]
 
-    # Let's try retrieve again, but Alice revoked the policy.
+    # FIXME: Bob tries to retrieve again
```

Comment:
Is there an issue for this?

Reply:
https://github.com/nucypher/nucypher/issues/833